apps/rush-lib/etc/examples/Sample_CustomCommandLine.json

```diff
@@ -7,19 +7,21 @@
 // see the documentation at https://aka.ms/rush
 
 {
+  "$schema": "https://dev.office.com/json-schemas/rush/command-line.schema.json",
   "customCommands": [
     {
       "name": "check-copyright",
       "summary": "Ensure that all files have a copyright header",
-      "documentation": "Ensures that all TS files have a copyright header"
+      "documentation": "Ensures that all TS files have a copyright header",
+      "parallelized": false
```

Comment:
>"parallelized": false [](start = 6, length = 21)

I thought this was the default now

Reply:
It is a required parameter

Reply:
I'll change it to true in the example

Reply:
This is the required parameter. I will change it to true.

---
In reply to: [148912952](https://github.com/Microsoft/web-build-tools/pull/431#discussion_r148912952) [](ancestors = 148912952)